### PR TITLE
Update libraries, and support SF 5.x

### PR DESCRIPTION
**Pull request by thePanz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,30 +15,19 @@ matrix:
...
     include:
-    - php: 7.1
-      env:
-      - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
-
     - php: 7.2
       env:
       - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
 
-    - php: '7.1'
```
Comment (dbu) [inline .travis.yml:26]: i am fine with dropping 7.1 support, but then we need to adjust the php version in the composer.json `require` section to `^7.2` as well

File: composer.json [hunk 2]
```diff
@@ -19,15 +19,15 @@
         "doctrine/annotations": "^1.6",
         "liip/metadata-parser": "^0.2",
         "pnz/json-exception": "^0.1.0",
```
Comment (Tobion) [inline composer.json:21]: I guess we'd also want to switch this to safe/ like in our projects.

Comment (dbu) [reply composer.json:21]: agree, it is the more commonly known library, even though the author of pnz/json-exception is well known :-D

Comment (thePanz) [reply composer.json:21]: I understand your reasons, but I would keep the current `pnz/json-exception` library.
My reason is the following: the `safe` library is pulling in 1.2Mb, of which we would use the JSON exception handling, while the `pnz` library is doing the same in only 48Kb

**Comment by thePanz** [comment]
@dbu branch alias adjusted to `1.1.x-dev`
Unfortunately there is no CHANGELOG in this repo :)

File: composer.json (hunk 2 above)
Comment (Tobion) [reply composer.json:21]: Do you plan to release a stable version of your library then?

Comment (Tobion) [reply composer.json:21]: Also I'm pretty sure we use other functions in liip/serializer that should use the safe alternative in theory.

Comment (Tobion) [reply composer.json:21]: https://github.com/thePanz/json-exception/issues/2 is blocking an update

**Comment by Tobion** [comment]
liip/serializer does not have any tags yet. For this reason changing the branch alias is pointless. I don't think we want to release 1.1 first.

File: composer.json (hunk 2 above)
Comment (Tobion) [reply composer.json:21]: @thePanz please update this to use your stable tag

**Pull request merged**